sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/convertor/CSVConvertor.java

```diff
@@ -0,0 +1,34 @@
+package com.exadel.sandbox.team5.service.convertor;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+public class CSVConvertor {
+
+    public ByteArrayInputStream createFile(Map<String, String> mapEntry, String statisticEntity, String amount) {
```

Comment:
Method can be static and class can me marked with @UtilityClass lombok annotation

Reply:
Done.